analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -54,8 +54,8 @@ import java.util.Stack
 /**
  * The Bower package manager for JavaScript, see https://bower.io/.
  */
-class Bower(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfiguration) :
-        PackageManager(analyzerConfig, repoConfig), CommandLineTool {
+class Bower(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfiguration) :
+        PackageManager(name, analyzerConfig, repoConfig), CommandLineTool {
```

Comment:
The naming is inconsistent, it's called `managerName` in `PackageManager`, but `name` here. Why not just `name` everywhere?

Reply:
Simply for shortness. I had it initially `name` everywhere, but was then running into naming clashes with local variables for some classes, so I've related the property to (the anyway better) `managerName`. In these constructor arguments, though, I felt this was not strictly necessary.

Reply:
Agreed.